package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Guardian Developers",
   "license": "MIT",
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=14.0.0"
```

Comment:
I'm wondering if we should remove this entirely? It [doesn't seem that relevant](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) in this context?